tag/update.go

```diff
@@ -28,6 +28,9 @@ func (r *ResolverForTag) UpdateTag(ctx context.Context, key string, newKey *stri
 	}
 
 	if newKey != nil && *newKey != key {
+		if strings.Contains(*newKey, " ") {
+			return nil, fmt.Errorf("tag must not contain spaces")
```

Comment:
Here the transaction must be closed otherwise it will block further requests. (already did the change)

Reply:
Sorry, missed that one.